.github/workflows/analyze-dependents.yml

```diff
@@ -227,7 +227,7 @@ jobs:
       with:
         node-version: ${{ env.NODE_VER }}
     - name: Install dependencies
-      run: npm ci
+      run: npm ci --ignore-scripts
```

Comment:
This will skip running `npm prepare` which is run during `npm ci` by default.

Reply:
The proposed fix here makes sense to me, though I'm wondering if you also considered exploring the recommendation in the `husky` documentation [here](https://typicode.github.io/husky/how-to.html#ci-server-and-docker):

> If installing only dependencies (not devDependencies), the "prepare": "husky" script may fail because Husky won't be installed. Modify the prepare script to never fail:

```
// package.json
"prepare": "husky || true"
```

Similar sanity-check type question: the `prepare` script in `package.json` includes `husky install`, which seems to differ a bit from just the `husky` command from their docs above. I do see we have v8 of `husky` installed when `v9` is latest so maybe this bit is due to the version discrepancy? 🤔

Reply:
Hmm, I haven't seen that part in their documentation, let me try that as well. And I didn't notice we are behind the latest version as well 😅 . (Thanks for the quick review!)

Reply:
I personally would like to see us just completely remove husky. I'm not a fan of git hooks - I find they get in the way when I just want to push something WIP up real quick (I know I can use `git push --no-verify` but still), and in general they slow down pushes. I also see them as redundant in our case considering the checks we have in CI.

That's definitely a conversation that is out of scope for this PR though, so if `husky || true` works I say :shipit:

Reply:
Yeah, I'm not a big fan of husky as well haha, feels like it causes more trouble than helps us

Reply:
Personally, I agree as well! I tend to use --no-verify out of habit in husky enabled repos lol.